app/src/main/res/layout-sw600dp-port/topic_info_fragment.xml

```diff
@@ -21,14 +21,17 @@
     <androidx.constraintlayout.widget.ConstraintLayout
       android:layout_width="match_parent"
       android:layout_height="wrap_content"
-      android:paddingBottom="24dp">
+      android:paddingBottom="112dp">
 
       <ImageView
         android:id="@+id/topic_thumbnail_image_view"
         android:layout_width="match_parent"
         android:layout_height="0dp"
         android:src="@{viewModel.topic.topicThumbnail.thumbnailGraphic}"
         android:background="@color/topicThumbnailBackground"
+        android:paddingTop="76dp"
+        android:paddingStart="98dp"
+        android:paddingEnd="98dp"
```

Comment:
Image should not contain any padding. The padding that you are seeing in the image in mocks, will actually be the part of image itself.

Reply:
Done